src/Process/Builder/ProcessBuilder.php

```diff
@@ -36,12 +36,13 @@ public function __construct(AbstractTestFrameworkAdapter $testFrameworkAdapter,
      * Creates process with enabled debugger as test framework is going to use in the code coverage.
      *
      * @param string $testFrameworkExtraOptions
+     * @param bool $skipCoverage
```

Comment:
we don't use useless PHPDoc `@param` since PHP7 already provides information *unless* there is a string, describing the method purpose as in this case `Creates process with enabled debugger as test framework is going to use in the code coverage.`

Without `@param` here, my IDE complains and underlines the method with warnings. I think this is a pretty common practice, and in this repository as well.